models/product.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  const product = sequelize.define('product', {
+    name: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    description: {
+      type: DataTypes.TEXT,
+      allowNull: false,
+    },
+    price: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    imageUrl: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+  }, {});
+  product.associate = function(models) {
+    product.belongsToMany(models.basket, { through: 'basketProducts',
+                                                    unique: false});
```

Comment:
this is some strange indentation. if you need to go to a new line you can go after opening the object, after the comma, or straight after opening the belongsToMany:
```
product.belongsToMany(models.basket, { 
      through: 'basketProducts',
      unique: false
});
```